lib/spack/docs/tutorial_modules.rst

```diff
@@ -653,88 +913,124 @@ and regenerating the module files for every package that depends on ``python``:
 
   proc ModulesHelp { } {
-  puts stderr "SciPy (pronounced "Sigh Pie") is a Scientific Library for Python. It"
+  puts stderr "SciPy (pronounced 'Sigh Pie') is a Scientific Library for Python. It"
   puts stderr "provides many user-friendly and efficient numerical routines such as"
   puts stderr "routines for numerical integration and optimization."
   }
 
-  if ![ is-loaded python/2.7.12-gcc-6.2.0 ] {
-      puts stderr "Autoloading python/2.7.12-gcc-6.2.0"
-      module load python/2.7.12-gcc-6.2.0
+  if ![ is-loaded python/2.7.14-gcc-7.2.0 ] {
+      puts stderr "Autoloading python/2.7.14-gcc-7.2.0"
+      module load python/2.7.14-gcc-7.2.0
   }
-
-  if ![ is-loaded openblas/0.2.19-gcc-6.2.0 ] {
-      puts stderr "Autoloading openblas/0.2.19-gcc-6.2.0"
-      module load openblas/0.2.19-gcc-6.2.0
+  if ![ is-loaded openblas/0.2.20-gcc-7.2.0 ] {
+      puts stderr "Autoloading openblas/0.2.20-gcc-7.2.0"
+      module load openblas/0.2.20-gcc-7.2.0
   }
-
-  if ![ is-loaded py-numpy/1.11.1-gcc-6.2.0-openblas ] {
-      puts stderr "Autoloading py-numpy/1.11.1-gcc-6.2.0-openblas"
-      module load py-numpy/1.11.1-gcc-6.2.0-openblas
+  if ![ is-loaded py-numpy/1.13.1-gcc-7.2.0-openblas ] {
+      puts stderr "Autoloading py-numpy/1.13.1-gcc-7.2.0-openblas"
+      module load py-numpy/1.13.1-gcc-7.2.0-openblas
   }
-
-  prepend-path CMAKE_PREFIX_PATH "~/spack/opt/spack/linux-Ubuntu14-x86_64/gcc-6.2.0/py-scipy-0.18.1-e6uljfiffgym4xvj6wveevqxfqnfb3gh/"
-  prepend-path LD_LIBRARY_PATH "~/spack/opt/spack/linux-Ubuntu14-x86_64/gcc-6.2.0/py-scipy-0.18.1-e6uljfiffgym4xvj6wveevqxfqnfb3gh/lib"
-  prepend-path PYTHONPATH "~/spack/opt/spack/linux-Ubuntu14-x86_64/gcc-6.2.0/py-scipy-0.18.1-e6uljfiffgym4xvj6wveevqxfqnfb3gh/lib/python2.7/site-packages"
-  setenv PY_SCIPY_ROOT "~/spack/opt/spack/linux-Ubuntu14-x86_64/gcc-6.2.0/py-scipy-0.18.1-e6uljfiffgym4xvj6wveevqxfqnfb3gh"
   conflict py-scipy
 
+  prepend-path LD_LIBRARY_PATH "/usr/local/opt/spack/linux-ubuntu16.04-x86_64/gcc-7.2.0/py-scipy-0.19.1-7hi7r5jri7bmohh4oontvfxo7rgj4hef/lib"
+  prepend-path CMAKE_PREFIX_PATH "/usr/local/opt/spack/linux-ubuntu16.04-x86_64/gcc-7.2.0/py-scipy-0.19.1-7hi7r5jri7bmohh4oontvfxo7rgj4hef/"
+  prepend-path PYTHONPATH "/usr/local/opt/spack/linux-ubuntu16.04-x86_64/gcc-7.2.0/py-scipy-0.19.1-7hi7r5jri7bmohh4oontvfxo7rgj4hef/lib/python2.7/site-packages"
+  setenv PY_SCIPY_ROOT "/usr/local/opt/spack/linux-ubuntu16.04-x86_64/gcc-7.2.0/py-scipy-0.19.1-7hi7r5jri7bmohh4oontvfxo7rgj4hef"
+
 and will contain code to autoload all the dependencies:
 
 .. code-block:: console
 
-  $ module load py-scipy
-  Autoloading python/2.7.12-gcc-6.2.0
-  Autoloading openblas/0.2.19-gcc-6.2.0
-  Autoloading py-numpy/1.11.1-gcc-6.2.0-openblas
+  root@module-file-tutorial:/# module load py-scipy
```

Comment:
I didn't get this output when I was following the instruction

Reply:
I was also able to follow this. Was your `modules.yaml` the same as the one shown a few lines above?

Reply:
yes, it was. But no output was shown.